### PR TITLE
fix link syntax on 4 links 

### PR DESCRIPTION
**Pull request by mrjones-plip**

### CODE REVIEW

File: content/en/design/icons/_index.md [hunk 1]
```diff
@@ -17,6 +17,6 @@ Use the `forms_tasks_targets` folder to find PNG and SVG icons for application c
 
 NOTE: For those using medic-conf to upload resources, make sure you add the correct icons for people and places to your `resources.json` file. Otherwise, uploading resources will inadvertently remove your people and place icons. An issue is being raised in medic-conf to solve this. You'll need to define `medic-clinic`, `medic-health-center`, `medic-district-hospital` and `medic-person` in your `resources.json` and set them to the appropriate SVG files.
 
-The design team has put together a resource to help with best practices when selecting and using icons in configuration. Check out the [best practices](../apps/) to learn more about which icons can be used for which use cases and workflows.
+The design team has put together a resource to help with best practices when selecting and using icons in configuration. Check out the [best practices]({{< ref "apps" >}}) to learn more about which icons can be used for which use cases and workflows.
```
Comment (abbyad) [inline content/en/design/icons/_index.md:20]: The link here should go to `design/apps/`

Comment (mrjones-plip) [reply content/en/design/icons/_index.md:20]: oh yeah! thanks so much for the 2nd set of eyes. I had merge conflict that I resolved and an earlier fix to this got blown away.  Really!! ;)

Comment (mrjones-plip) [reply content/en/design/icons/_index.md]: I think more specifically it should be `design/apps#icons`, yeah?  should be fixed in 02ec115

**Comment by abbyad** [comment]
@mrjones-plip is this PR ready waiting on more work?

**Comment by mrjones-plip** [comment]
@abbyad  - no, it's not! I just forgot to request another review, and then closed the ticket for some reason.  

lemme know if it looks good with mys last commit.  thanks for the follow up!

**Pull request merged**